packages/dds/tree/src/feature-libraries/default-schema/defaultFieldKinds.ts

```diff
@@ -59,16 +60,17 @@ const optionalIdentifier = "Optional";
 /**
  * 0 or 1 items.
  */
-export const optional = new FieldKindWithEditor(
-	optionalIdentifier,
-	Multiplicity.Optional,
-	optionalChangeHandler,
-	(types, other) =>
-		(other.kind.identifier === sequence.identifier ||
-			other.kind.identifier === optionalIdentifier) &&
-		allowsTreeSchemaIdentifierSuperset(types, other.types),
-	new Set([]),
-);
+export const optional: FieldKindWithEditor<OptionalFieldEditor, Multiplicity.Optional, "Optional"> =
```

Comment:
Some explicit types added where they were simple, to reduce the number of dynamic imports in `d.?ts` files (same below in this file).